examples/digital_in/cont_acq_dig_lines_int_clk.py

```diff
@@ -0,0 +1,28 @@
+﻿"""Example for reading digital signals.
+
+This example demonstrates how to acquire a continuous digital
+waveform using the DAQ device's internal clock.
+"""
+
+import nidaqmx
+from nidaqmx.constants import AcquisitionType, LineGrouping
+
+
+with nidaqmx.Task() as task:
+    task.di_channels.add_di_chan("Dev1/port0/line0:3", line_grouping=LineGrouping.CHAN_PER_LINE)
+    task.timing.cfg_samp_clk_timing(1000.0, sample_mode=AcquisitionType.CONTINUOUS)
+    task.start()
+    print("Acquiring samples continuously. Press Ctrl+C to stop.")
+
+    try:
+        total_read = 0
+        while True:
+            data = task.read(number_of_samples_per_channel=4)
```

Comment:
(1000 S/s) / (4 S/read) = 250 read/s which is too fast. The reason I suggested passing this parameter was to avoid this situation.

The C example ContReadDigChan-ExtClk.c acquires at 10000 S/s and reads 1000 S/read, so it does 10 read/s which is fine. Doing 1 read/s is also fine.

Reply:
BTW, "too fast" is subjective, but some considerations:
- A PCIe device can easily keep up with 250 reads/sec, but a USB or Ethernet device may not
- A typical PC display updates 50 or 60 times/sec, so updating the screen 250 times/sec is overdoing it